thunder/core/prims.py

```diff
@@ -1945,9 +1945,12 @@ def _numpy_array_to_torch_tensor_meta(a: TensorProxy, /) -> TensorProxy:
 #   usually produce an output with that same datatype (SAME).
 #   Sometimes, however, elementwise operations can produce an output with a different
 #   datatype than the inputs. For example, comparison operations like eq and lt always
-#   produce boolean results (ALWAYS_BOOL), math.ceil/math.floor produces integer outputs for number inputs while preserves datatype for tensor inputs, and other operations, like abs, map
-#   complex numbers to floats (COMPLEX_TO_FLOAT).
+#   produce boolean results (ALWAYS_BOOL), ceil/floor produces integer outputs for
+#   number inputs while preserves datatype for tensor inputs (INT_FOR_NUMBER), and
+#   other operations, like abs, map complex numbers to floats (COMPLEX_TO_FLOAT).
 #   The ELEMENTWISE_PRIM_OUTPUT_DTYPE_KIND enum describes these three behaviors so that
```

Comment:
This line is redundant with the `The ELEMENTWISE_PRIM_OUTPUT_DTYPE_KIND enum describes these four behaviors so that` below.